images/upload-gitlab-failure-logs/taxonomy.yaml

```diff
@@ -2,6 +2,21 @@
 taxonomy:
   version: "0.0.0-dev"
   error_classes:
+    network_timeout:
+      grep_for:
+        - "socket.timeout"
+        - "curl: (28)"
```

Comment:
I think the regexes need testing, I'm fairly certain @mvandenburgh recently fixed some issues with unescaped parens in other regexes, so I think these aren't going to work as-is.

Reply:
Yup, these will need to be escaped. https://github.com/spack/spack-infrastructure/pull/417

Reply:
yeah, no testing on this yet. Is the `grep_for` pattern being passed to a regular `grep "my cool pattern"` then?